### PR TITLE
Add support for model scores endpoint

### PR DESCRIPTION
**Pull request by christophe-williams**

### CODE REVIEW

File: hudai/resources/model_score.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        return self._fetch(entity_id)
+
+    def upsert(self,
+               content_id=None,
+               content_type=None,
+               importance_score=None,
+               tipsheet_X_G_B_score=None,
+               local_X_G_B_score=None,
+               shill_X_G_B_score=None):
+        return self._create(
```
Comment (chriswynnyk) [inline hudai/resources/model_score.py]: `self._upsert(` ?

Comment (christophe-williams) [reply hudai/resources/model_score.py]: I created that method in client.py - makes a put request without the entity_id. I could just bring that function into this class instead. There are other upsert endpoints but looks like they're exposed through the js client only.

Comment (chriswynnyk) [reply hudai/resources/model_score.py]: I'm okay with/without defining an upsert...  Just  a little unclear on how this function is distinct from create?  
(It's also a code smell to be named one thing and do another)

Comment (christophe-williams) [reply hudai/resources/model_score.py]: It creates if the content_id doesn't exist, and updates if it does

Comment (christophe-williams) [reply hudai/resources/model_score.py]: Got the issue - changing now

**Pull request merged**